packages/api-genesis/schema.graphql

```diff
@@ -3,15 +3,60 @@ schema {
 }
 
 scalar JSONObject
+scalar Timestamp
 
 type Query {
   genesis: Genesis!
 }
 
 type Genesis {
+  byron: ByronGenesis
   shelley: ShelleyGenesis
 }
 
+type ByronBlockVersionData {
+  scriptVersion: Int!
+  slotDuration: Int!
+  maxBlockSize: Int!
+  maxHeaderSize: Int!
+  maxTxSize: Int!
+  maxProposalSize: Int!
+  mpcThd: String!
+  heavyDelThd: String!
+  updateVoteThd: String!
+  updateProposalThd: String!
+  updateImplicit: String!
+  softforkRule: ByronSoftForkRule!
+  txFeePolicy: ByronTxFeePolicy!
+  unlockStakeEpoch: String!
+}
+
+type ByronGenesis {
+  bootStakeholders: JSONObject!
+  heavyDelegation: JSONObject!
+  startTime: Timestamp!
+  nonAvvmBalances: JSONObject!
+  blockVersionData: ByronBlockVersionData!
+  protocolConsts: ByronProtocolConsts!
```

Comment:
Could this be named `protocolParams` to match the naming convention of `ShelleyGenesis` too?